docs/docusaurus.config.js

```diff
@@ -44,6 +44,14 @@ const config = {
   themeConfig:
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
     ({
+      announcementBar: {
+        id: 'experimental',
+        content:
+          '🚧  This is an experimental framework. Do not use in production at this time. 🚧 ',
```

Comment:
Isn't `Do not use in production at this time.` too strong here? Maybe something like `use at your own risk` or something like that?

Reply:
I just kept the previous copy but happy to adjust. This is going to go away soon anyway hopefully 😄

Reply:
Resolved 👍